rdf-prefix.el

```diff
@@ -202,8 +212,7 @@
     ("bf" . "http://id.loc.gov/ontologies/bibframe/")
     ("bflc" . "http://id.loc.gov/ontologies/bflc/")
     ("bfo" . "http://purl.obolibrary.org/obo/")
-    Wide character in print at rdf-prefix-update.pl line 20.
-    ("bgcat" . "http://bg.dbpedia.org/resource/Категория:")
+    ("bgcat" . "http://bg.dbpedia.org/resource/ÐšÐ°Ñ‚ÐµÐ³Ð¾Ñ€Ð¸Ñ:")
```

Comment:
You can delete this prefix. It belongs to me, and I'll delete it from prefix.cc (it's not used)